<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>空文件夹扫描</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="lineEditPath"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonPath">
               <property name="text">
                <string>...</string>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>路径</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>镜头名正则</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditShotPattern"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>空文件夹</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonRefresh">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="listView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QToolButton" name="toolButtonTxt">
             <property name="text">
              <string>生成txt</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QLabel" name="labelVersion">
             <property name="text">
              <string>version</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAskPath">
   <property name="text">
    <string>AskPath</string>
   </property>
  </action>
  <action name="actionTxt">
   <property name="text">
    <string>txt</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonPath</sender>
   <signal>clicked()</signal>
   <receiver>actionAskPath</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonTxt</sender>
   <signal>clicked()</signal>
   <receiver>actionTxt</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRefresh</sender>
   <signal>clicked()</signal>
   <receiver>actionRefresh</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPath</sender>
   <signal>editingFinished()</signal>
   <receiver>actionRefresh</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditShotPattern</sender>
   <signal>editingFinished()</signal>
   <receiver>actionRefresh</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
